Networking/NetworkingTests/Mapper/SitePluginMapperTests.swift

```diff
@@ -0,0 +1,45 @@
+import XCTest
+@testable import Networking
+
+/// SitePluginMapper Unit Tests
+///
+class SitePluginMapperTests: XCTestCase {
```

Comment:
nit: we usually mark a class as final if we don't plan on subclassing it 🙂 
```suggestion
final class SitePluginMapperTests: XCTestCase {
```